charts/airbyte-temporal/templates/_helpers.tpl

```diff
@@ -30,10 +30,30 @@ Create chart name and version as used by the chart label.
 {{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" }}
 {{- end }}
 
+{{/*
+Common labels
+*/}}
+{{- define "airbyte.labels" -}}
+helm.sh/chart: {{ include "airbyte.chart" . }}
+{{ include "airbyte.selectorLabels" . }}
+{{- if .Chart.AppVersion }}
+app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
+{{- end }}
+app.kubernetes.io/managed-by: {{ .Release.Service }}
+{{- end }}
+
+{{/*
+Selector labels
+*/}}
+{{- define "airbyte.selectorLabels" -}}
+app.kubernetes.io/name: {{ include "airbyte.name" . }}
+app.kubernetes.io/instance: {{ .Release.Name }}
+{{- end }}
```

Comment:
Curious, what does this render to?

Is it:
airbyte: temporal?

Reply:
No, it'll be 
```
app.kubernetes.io/name: temporal
```
It's a proposed changes from a contributor in #15957 to make helm remove all the stuff it deployed(it works)

Reply:
understand.

Reply:
as long as this is ok in Cloud.